src/patient-registration/patient-registration.test.tsx

```diff
@@ -33,7 +34,7 @@ describe('demographics section', () => {
   it('updates to correct gender', async () => {
     const { container } = render(<PatientRegistration />);
     const genderSelect = container.querySelector('select[name="gender"]') as HTMLSelectElement;
-    const expectedGender = 'M';
+    const expectedGender = 'Male';
```

Comment:
See previous comments regarding this. I believe this is also out of scope for this ticket.

Reply:
We kept the selectInput general and for the gender we make sure that the payload contains only the first letter.